tests/or_lookups/tests.py

```diff
@@ -234,3 +234,11 @@ def test_other_arg_queries(self):
             Article.objects.filter(Q(headline__startswith='Hello')).in_bulk([self.a1, self.a2]),
             {self.a1: Article.objects.get(pk=self.a1)}
         )
+
+        self.assertEqual(
```

Comment:
Does this test cover any changed code that the other test doesn't cover? If not, I'd omit it.